src/phoenix-adapter.js

```diff
@@ -0,0 +1,258 @@
+import { transportForChannel } from "./transport-for-channel";
+import { authorizeOrSanitizeMessage } from "./utils/permissions-utils";
+
+// TODO: Use the websocket connection, not HEAD requests
+const getTimeOffsetToServer = async () => {
```

Comment:
This is the same algorithm for computing the time offset to the server as was in `naf-dialog-adapter`.